terms.html

```diff
@@ -5,12 +5,18 @@
 <dl class="termlist">
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made about an <a>entity</a> by a specific <a>entity</a>.
+The entity about which the assertion is made is called the <a>subject</a> of the claim.
+The meaning of the claim (its semantics) is determined by its author.
+Claims can be authored by <a>issuers</a>, e.g., to construct a <a>verifiable credential</a> with, or 
+by <a>holders</a>, e.g., to include in a <a>verifiable presentation</a>.
+Different authors can reuse the same semantics for claims through the use of shared vocabularies.
```

Comment:
```suggestion
issuers and verifiers can use the same semantics for claims through the use of shared vocabularies.
```

Reply:
Corrects capitalization, adds links to defined terms, and adds a few words to increase clarity.
```suggestion
<a>Issuers</a> and <a>verifiers</a> can use the same semantics for claims through the use of shared vocabularies and/or ontologies.
```